patroni/ctl.py

```diff
@@ -1092,12 +1092,28 @@ class opts:
             width = 80
             tab_width = 8
             wrap = True
-            if shutil.which('less'):
-                pager = None
-            else:
-                pager = os.path.basename(shutil.which('more') or 'more')
+            pager = next(
+                (
+                    os.path.basename(p)
+                    for p in (os.environ.get('PAGER', None), "less", "more")
```

Comment:
`dict.get` returns `None` by default if value is missing
```suggestion
                    for p in (os.environ.get('PAGER'), "less", "more")
```

Reply:
I missed changing the unit test, doing that now

Reply:
done!